pilot/pkg/kube/inject/inject_test.go

```diff
@@ -88,7 +88,6 @@ func TestIntoResourceFile(t *testing.T) {
 		{
 			in:                           "hello.yaml",
 			want:                         "hello.yaml.injected",
-			debugMode:                    true,
```

Comment:
Does this mean we don't have a debugMode option in helm template to control the Docker image used?

Reply:
Yes. We do not hava option `debugMode` in helm template. Do we need to add one?

Reply:
It'll be nice if you can add one, but also totally fine to do it later as separate PR.